src/MultiplicationTable.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner;
+
+public class MultiplicationTable {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        int amountColumns = scanner.nextInt();
+        int amountLines = scanner.nextInt();
```

Comment:
Are the names consistent with what you are trying to do? Shouldn't it be lines (or rows) first and then columns?

Reply:
Why did you resolve this? Are you saying this is correct already? I don't see anything fixed